ruff.toml

```
@@ -6,8 +6,25 @@ extend-select = [
 	"C901",
 	"PERF401",
 	"W",
+ 
+ 	# Ensure modern type annotation syntax and best practices
+	# Not including those covered by type-checkers or exclusive to Python 3.11+
+	"FA", # flake8-future-annotations
+	"F404", # late-future-import
+	"PYI", # flake8-pyi
+	"UP006", # non-pep585-annotation
+	"UP007", # non-pep604-annotation
+	"UP010", # unnecessary-future-import
+	"UP035", # deprecated-import
+	"UP037", # quoted-annotation
+	"UP043", # unnecessary-default-type-args
+
+	# local
 ]
 ignore = [
+	# Typeshed doesn't want complex or non-literal defaults for maintenance and testing reasons.
+	# This doesn't affect us, let's have more complete stubs.
```

Let's see if we can rephrase while honoring three grammatical nitpicks I bear.

1. Avoid first- and second-person.
2. Always stipulate the subject when using "this" (i.e. "this (what?)").
3. Avoid comma-splice.

```suggestion
	# Typeshed rejects complex or non-literal defaults for maintenance and testing reasons,
	# irrelevant to this project.
```